Source/Scene/modelMaterialsCommon.js

```diff
@@ -237,6 +237,11 @@ define([
             }
         }
 
+        // Give the diffuse uniform a semantic to support color replacement in 3D Tiles
+        if (defined(techniqueParameters.diffuse)) {
```

Comment:
This is backwards compatibility, right?  Can we mention that in the comment?

I am also tempted to make this more robust, e.g., check if another semantic is already `_3DTILESDIFFUSE` and if this already has a semantic.  Maybe we should since this parameter name is potentially common and not just from COLLADA2GLTF.